operator/config/manager/kustomization.yaml

```diff
@@ -8,5 +8,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
```

Comment:
can we move this out of operator and into a top level folder like `develop`

Reply:
I will tackel this in the next PR with make file changes